setup.py

```diff
@@ -45,7 +45,7 @@
 TESTS_REQUIRE = ['pytest', 'pytest-cov', 'pytest-mock', 'nose']
 EXTRAS_REQUIRE = {
     'optional': ['scipy', 'tables', 'numba', 'siphon', 'netcdf4',
-                 'ephem', 'cython'],
+                 'ephem', 'cython', 'pvfactors==0.1.5'],
```

Comment:
Thanks @wholmgren! Understood, I'll remove it. What is the best way to let the users know that they need pvfactors v0.1.5 for this pvlib implementation to work? And that later pvfactors versions may have updated APIs that would break this implementation?

Reply:
> What is the best way to let the users know that they need pvfactors v0.1.5 for this pvlib implementation to work? 

Would a `>=` requirement work? Are you planning on breaking this functionality in an 0.2 release in the near future? 

> And that later pvfactors versions may have updated APIs that would break this implementation?

I'm hoping that you'll keep the pvlib wrappers up to date!

Reply:
Thanks! The only user request I got on that was to be able to provide timeseries albedo, but I can always make it backwards compatible